doc/Quick_start/deployment.md

```diff
@@ -23,9 +41,9 @@
     * More precise explanation of the required fields:
         * For Windows computes use `bms_win` dict instead of regular `bms`
         * Roles supported on Windows are `vrouter` and `win_docker_driver` - specify them
-        * Set `WINDOWS_PHYSICAL_INTERFACE` to dataplane interface alias
-        * Specify `WINDOWS_ENABLE_TEST_SIGNING` if you want to install untrusted artifacts (this is needed at this moment)
-        * Set `WINDOWS_DEBUG_DLLS_PATH` to path on Ansible machine containing MSVC 2015 debug dlls, specifically: `msvcp140d.dll`, `ucrtbased.dll` and `vcruntime140d.dll`
+        * Set `WINDOWS_PHYSICAL_INTERFACE` to dataplane interface name (run `Get-NetAdapter` from PowerShell to list available interfaces on Windows compute node)
+        * Add `WINDOWS_ENABLE_TEST_SIGNING` option and leave it empty, if you want to install untrusted artifacts (this is needed at this moment)
```

Comment:
We should probably have a separate section explaining two optional steps, because it's not clear atm:

1. certificates - testing and official, how to deploy in both configurations, what is needed, what it means etc.
2. dlls - debug vs release builds.